lib/src/widgets/primitives/box.dart

```diff
@@ -82,17 +82,35 @@ class BoxMixerWidget extends MixerWidget {
     }
 
     if (mixer.backgroundColor != null) {
-      current = ColoredBox(
-        color: mixer.backgroundColor!.value,
-        child: current,
-      );
+      if (mixer.backgroundColor!.hasAnimation) {
+        current = AnimatedContainer(
+          duration: mixer.backgroundColor!.animationDuration!,
+          curve: mixer.backgroundColor!.animationCurve!,
+          color: mixer.backgroundColor!.value,
+          child: current,
+        );
+      } else {
+        current = ColoredBox(
+          color: mixer.backgroundColor!.value,
+          child: current,
+        );
+      }
```

Comment:
I didn't find a way to animate solely the `ColoredBox`. We might want to create a widget that animates animated mixes (`AnimatedColoredBox` and `AnimatedDecoratedBox`, for example).